<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_sign</class>
 <widget class="QDialog" name="dlg_sign">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>签到</string>
  </property>
  <widget class="QLabel" name="lbl_info">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>400</width>
     <height>180</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>1、重置日常任务次数。
2、金币x万</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_total">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 127, 19);</string>
   </property>
   <property name="text">
    <string>连续签到天数：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>签到</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_retry">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>换个服务器签到</string>
   </property>
  </widget>
  <widget class="QLabel" name="_bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>350</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="pixmap">
    <pixmap resource="mirror.qrc">:/mirror/Resources/ui/31.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>35</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(180, 0, 0);</string>
   </property>
   <property name="text">
    <string>签到奖励：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="lbl_status">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>400</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>_bg</zorder>
  <zorder>lbl_info</zorder>
  <zorder>lbl_total</zorder>
  <zorder>btn_ok</zorder>
  <zorder>btn_retry</zorder>
  <zorder>label_4</zorder>
  <zorder>lbl_status</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mirror.qrc"/>
 </resources>
 <connections/>
</ui>
